### PR TITLE
fix: Add timeout option to the `invoke` and `its` types

### LINKED ISSUE #9079: Add timeout parameter type to cy.invoke and cy.its commands
Cypress v5.5.0

Developers will be able to add a timeout to the invoke and its commands.

The cypress documentation lists timeout as one of the available
options.

See:

- docs.cypress.io/api/commands/invoke.html#Arguments
- docs.cypress.io/api/commands/its.html#Arguments

### PR DESCRIPTION
**Pull request by davidmh**
- closes #9079 

### User facing changelog

Developers will be able to add a timeout to the `invoke` and `its` commands.

The cypress documentation lists `timeout` as one of the available
options.

See:
- https://docs.cypress.io/api/commands/invoke.html#Arguments
- https://docs.cypress.io/api/commands/its.html#Arguments

### PR Tasks

- [x] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [x] add types tests
- [x] marked issue release in ZenHub

### CODE REVIEW

File: cli/types/cypress.d.ts [hunk 1]
```diff
@@ -1107,7 +1107,7 @@ declare namespace Cypress {
       ...args: any[]
     ): Chainable<R>
     invoke<K extends keyof Subject, F extends ((...args: any[]) => any) & Subject[K], R = ReturnType<F>>(
-      options: Loggable,
+      options: Loggable | Timeoutable | Loggable & Timeoutable,
```
Comment (bahmutov) [inline cli/types/cypress.d.ts:1110]:
```suggestion
      options: Partial<Loggable & Timeoutable>,
```

**Comment by bahmutov** [comment]
Please also add a type test with timeout option in https://github.com/cypress-io/cypress/blob/develop/cli/types/tests/cypress-tests.ts#L133 please?

**Comment by jennifer-shehane** [comment]
@davidmh Do you still have time to address the changes requested in this PR? We will have to close this PR due to inactivity otherwise.

**Comment by davidmh** [comment]
@jennifer-shehane thank you for the ping, the request for changes got lost in my notifications.

@bahmutov I've updated the PR with the type tests for `cy.invoke` and the same update for `cy.its`

**Review by bahmutov — APPROVED** [review]
super, thank you very much

**Pull request merged**